ios/FluentUI/Button/Button.swift

```diff
@@ -53,83 +53,48 @@ open class Button: UIButton, Shadowable, TokenizedControlInternal {
         }
     }
 
+    /// The content insets of the buton.
+    /// If these insets are not equal to `defaultEdgeInsets()`, the minimum height constraint will be disabled.
     open lazy var edgeInsets: NSDirectionalEdgeInsets = defaultEdgeInsets() {
         didSet {
             isUsingCustomContentEdgeInsets = edgeInsets != defaultEdgeInsets()
+            minHeightConstraint.isActive = !isUsingCustomContentEdgeInsets
 
             invalidateIntrinsicContentSize()
 
             if !isAdjustingCustomContentEdgeInsetsForImage && image(for: .normal) != nil {
                 adjustCustomContentEdgeInsetsForImage()
             }
 
-            if #available(iOS 15.0, *) {
-                var configuration = self.configuration ?? UIButton.Configuration.plain()
-                configuration.contentInsets = edgeInsets
-                self.configuration = configuration
-            } else {
-                let left: CGFloat
-                let right: CGFloat
-                if effectiveUserInterfaceLayoutDirection == .leftToRight {
-                    left = edgeInsets.leading
-                    right = edgeInsets.trailing
-                } else {
```

Comment:
Do we not need to handle different insets for RTL anymore?

Reply:
We're using `NSDirectionalEdgeInsets` now, which uses leading + trailing, instead of `UIEdgeInset`, which used left + right, so RTL support is built in.